docs/index.md

```diff
@@ -7,7 +7,7 @@ description: |-
 
 # CDP Provider
 
-The Cloudera Data Platform (CDP) provider manages the lifecycle of resources supported by CDP like Credentials, Environment, Datalake, Datahub and other data services.
+The Cloudera Data Platform (CDP) provider manages the lifecycle of resources supported by CDP like Credentials, Environment, Datalake, Datahub and other data services. The provider currently includes the resources required to set up CDP on AWS and Azure.
```

Comment:
This file is templated, so it will be overriden the next time anybody runs: 
```
make docs
```

Can you please make the changes to `templates/index.md.tmpl` instead?

Reply:
Done. I also ran `make docs` and include the generated docs in this PR.